diagnostic.md

```diff
@@ -5,20 +5,21 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    you could have a list that has a component of items.
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember g component my-map
     ```
 
 1.  What files are edited to produce a component, and what are their
     responsibilities?
 
     ```md
-    <!-- your response here -->
+    component.js - is responsible for assigning the proper actions and styling defaults for the generated html/css
+    template.js  - is responsible for rendering the data for the generated component.
```

Comment:
Looks good.  The `template.hbs` is also responsible for binding and dispatching actions.